conf/test.config

```
@@ -32,4 +32,8 @@ params {
 
     // diamond_blastp
     diamondblastp_db = '/lustre/scratch123/tol/teams/tolit/users/ar34/diamond_reduced/mMelMel3.1.buscogenes.dmnd'
+
+    // GCA accesion or path to YAML file
+    accesion = 'GCA_922984935.2'
```

Same typo in many other files
```suggestion
    accession = 'GCA_922984935.2'
```